### PR TITLE
Replace hethers.js with ethers.js in hardhat-example

### PR DESCRIPTION
**Pull request by georgi-l95**
Signed-off-by: georgi-l95 <glazarov95@gmail.com>

**Description**:
Replace hethers.js with ethers.js in our hardhat example in tools folder.
Bump network and mirror node images in tests

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: tools/hardhat-example/hardhat.config.js [hunk 1]
```diff
@@ -46,12 +28,31 @@ task('contract-call', async (taskArgs) => {
...
-    }
-  }
+    local: {
+      url: process.env.RELAY_ENDPOINT,
+      accounts: [process.env.OPERATOR_PRIVATE_KEY, process.env.RECEIVER_PRIVATE_KEY],
+      chainId: 298,
+    },
+    testnet: {
+      url: 'https://testnet.hashio.io/api',
+      accounts: [process.env.OPERATOR_PRIVATE_KEY_TESTNET, process.env.RECEIVER_PRIVATE_KEY_TESTNET],
```
Comment (Nana-EC) [inline tools/hardhat-example/hardhat.config.js:54]: Leave this array empty.
Let's not add another config just for testnet. Let devs customize this part or reuse the same config name with the intent they'd fill it in appropriately as needed

File: tools/hardhat-example/hardhat.config.js [hunk 2]
```diff
@@ -1,34 +1,16 @@
-/*-
```
Comment (Nana-EC) [inline tools/hardhat-example/hardhat.config.js:1]: Restore license header

File: tools/hardhat-example/README.md [hunk 3]
```diff
@@ -3,22 +3,27 @@
...
 ## Configuration
 
 Create `.env` file based on `.env.example`
+
 ```
 # Alias accounts keys
 OPERATOR_PRIVATE_KEY=
 RECEIVER_PRIVATE_KEY=
-
+OPERATOR_PRIVATE_KEY_TESTNET=
```
Comment (Nana-EC) [inline tools/hardhat-example/README.md:19]: See comment below. I don't think we need to add another config variable

**Review by Nana-EC — COMMENTED** [review]
Looking good on actual changes.
Some bits on license and the new testnet config

File: tools/hardhat-example/hardhat.config.js (hunk 1 above)
Comment (georgi-l95) [reply tools/hardhat-example/hardhat.config.js]: You're right. Left it empty array.

File: tools/hardhat-example/hardhat.config.js (hunk 2 above)
Comment (georgi-l95) [reply tools/hardhat-example/hardhat.config.js]: Missed that one. Restored.

File: tools/hardhat-example/README.md (hunk 3 above)
Comment (georgi-l95) [reply tools/hardhat-example/README.md]: Removed it.

File: tools/hardhat-example/hardhat.config.js (hunk 2 above)
Comment (georgi-l95) [reply tools/hardhat-example/hardhat.config.js]: Missed that one. Restored

**Review by natanasow — APPROVED** [review]
LGTM

File: tools/hardhat-example/scripts/deployContract.js [hunk 4]
```diff
@@ -18,18 +18,14 @@
...
+const { ethers } = require('hardhat');
 
 module.exports = async () => {
-  const provider = new hre.ethers.providers.JsonRpcProvider(process.env.RELAY_ENDPOINT);
-  const wallet = new hre.ethers.Wallet(process.env.OPERATOR_PRIVATE_KEY, provider);
-  const Greeter = await hre.ethers.getContractFactory('Greeter', wallet);
-  const greeter = await Greeter.deploy('initial_msg');
+  let wallet = (await ethers.getSigners())[0];
+  const Greeter = await ethers.getContractFactory('Greeter', wallet);
+  const greeter = await Greeter.deploy('initial_msg', { gasLimit: 1000000 });
```
Comment (Nana-EC) [inline tools/hardhat-example/scripts/deployContract.js:26]: Why the explicit gasLimit here and above?
I sit just to indicate such calls don't cost more or is it required now for some reason?
If this needs to be explicit we should be clear why with a comment or some kind of an issue etc

**Review by Nana-EC — APPROVED** [review]
LG
Curious on the gasLimit detail

File: tools/hardhat-example/scripts/deployContract.js (hunk 4 above)
Comment (georgi-l95) [reply tools/hardhat-example/scripts/deployContract.js:26]: Actually, it's not needed. I've tested and probably forgot to remove it. Will do it now. Also ethers will automatically determine the gas, no need for explicitly writing the gasLimit.

**Review by natanasow — APPROVED** [review]
LGTM

**Pull request merged**